web/templates/inbox_view/inbox_row.hbs

```diff
@@ -48,19 +48,19 @@
                 <div class="inbox-right-part">
                     {{#if is_topic}}
                         <span class="visibility-policy-indicator change_visibility_policy hidden-for-spectators{{#if (eq visibility_policy all_visibility_policies.INHERIT)}} inbox-row-visibility-policy-inherit{{/if}}"
-                          data-stream-id="{{stream_id}}" data-topic-name="{{topic_name}}" tabindex="0" data-col-index="{{ column_indexes.TOPIC_VISIBILITY }}">
+                          data-stream-id="{{stream_id}}" data-stream-name="{{stream_name}}" data-topic-name="{{topic_name}}" tabindex="0" data-col-index="{{ column_indexes.TOPIC_VISIBILITY }}">
                             {{#if (eq visibility_policy all_visibility_policies.FOLLOWED)}}
-                                <i class="zulip-icon zulip-icon-follow recipient_bar_icon" data-tippy-content="{{t 'You follow this topic'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You follow this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-follow recipient_bar_icon" data-tippy-content="{{t 'You follow this topic.'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'You follow this topic.' }}"></i>
                             {{else if (eq visibility_policy all_visibility_policies.UNMUTED)}}
-                                <i class="zulip-icon zulip-icon-unmute-new recipient_bar_icon" data-tippy-content="{{t 'You have unmuted this topic'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You have unmuted this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-unmute-new recipient_bar_icon" data-tippy-content="{{t 'You have unmuted this topic.'}}"
+                                  role="button" aria-haspopup="true" aria-label="{{t 'You have unmuted this topic.' }}"></i>
```

Comment:
Let's avoid using period  for the aria labels when there is no sentence following it. It would be read aloud and not serve much purpose.